src/helpers/geoChart/colorMapping.ts

```diff
@@ -0,0 +1,44 @@
+// (C) 2020 GoodData Corporation
```

Comment:
This file name should be `colorStrategy.ts`

Reply:
fixed